openeo/rest/datacube.py

```diff
@@ -2860,3 +2804,71 @@ def unflatten_dimension(self, dimension: str, target_dimensions: List[str], labe
                 label_separator=label_separator,
             ),
         )
+def _get_geometry_argument(
```

Comment:
in the code style category: you would need two empty lines before starting a new function